app/src/main/java/id/rizmaulana/covid19/data/model/CovidOverview.kt

```diff
@@ -7,7 +7,7 @@ import com.google.gson.annotations.SerializedName
  * rizmaulana 04/03/20.
  */
 data class CovidOverview(
-    @Expose @SerializedName("confirmed") val confirmed: CovidOverviewItem? = null,
-    @Expose @SerializedName("recovered") val recovered: CovidOverviewItem? = null,
-    @Expose @SerializedName("deaths") val deaths: CovidOverviewItem? = null
+    @Expose @SerializedName("confirmed") val confirmed: CovidOverviewItem = CovidOverviewItem(),
```

Comment:
Refer to https://github.com/rizmaulana/kotlin-mvvm-covid19/pull/2#discussion_r390191156